tools/spades_header_fixer/spades_header_fixer.xml

```diff
@@ -0,0 +1,20 @@
+<tool id="spades_header_fixer" name="SPAdes fasta header fixer" version="1.1.1">
+	<description>Fixes Spades Fasta ids</description>
+	<command >sed -r "s/>NODE(_[0-9]+)_(.*)/>${input.name}\1 \2/g" $input >$output</command>
```

Comment:
switch to **input.element_identifier** instead of **input.name** Since this tool was written, using element_identifier works better on collections

Reply:
Done, fixed in commit 2946ebc